packages/create-bison-app/template/tests/unit/components/LoginForm.test.tsx

```diff
@@ -1,43 +0,0 @@
-/**
- * @jest-environment jsdom
- */
-
-import { render, waitFor, userEvent } from '@/tests/utils';
-import { LoginForm } from '@/components/auth/LoginForm';
-
-describe('LoginForm', () => {
-  describe('with no inputs', () => {
-    it('shows errors', async () => {
-      const { getByLabelText, getByText, getAllByLabelText } = render(<LoginForm />);
-
-      const submitButton = getByText(/sign in/i);
-      userEvent.click(submitButton);
-
-      const emailInput = getByLabelText(/email/i);
-      const passwordInput = getAllByLabelText(/password/i)[0];
-
-      await waitFor(() => expect(emailInput).toHaveAttribute('aria-invalid', 'true'));
```

Comment:
Because email and password are required fields, we never get far enough for react-hook-form to actually validate the form, and `area-invalid` is never set. So the tests in this file can't ever pass.